ocs_ci/ocs/fdf_upgrade.py

```
@@ -159,6 +162,36 @@ def load_version_config_file(self, upgrade_version):
+            CSVNotFound: If the CSV with the provided resource isn't found.
+
+        Returns:
+            str: Name of the resource CSV
+
+        """
+        version = self.version_before_upgrade
+        csv_list = get_csvs_start_with_prefix(resource_name, namespace=self.namespace)
+        for csv in csv_list:
+            csv_name = csv.get("metadata").get("name")
+            if resource_name in csv_name:
+                logger.info(f"Searching for pre-upgrade CSV with version: {version}")
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)